libs/utils.liq

```diff
@@ -513,6 +502,16 @@ def stereo.right(s)
   mean(stereo.pan(pan=1., s))
 end
 
+# Append speech-synthesized tracks reading the metadata.
+# @category Source / Track Processing
+# @param ~pattern Pattern to use
+# @param s The source to use
+def say_metadata
+  p = 'say:$(if $(artist),"It was $(artist)$(if $(title),\\", $(title)\\").")'
```

Comment:
We need to remove those interpolation patterns one day..

Reply:
Yes...